boscoseries/iReporter
# (bug) travis: implement feedback on travisCI [fixes #162445783]

### Description of Task to be completed?
setup travisCI for continous integration
### How should this be manually tested?
clone the repository
checkout into branch bug-server-setup-travisCI-162445783
run npm test and monitor travis build.
this should pass build
### What are the relevant pivotal tracker stories?
#162445783
